### ISSUE TITLE
Carousel component design specs - Target:  Jan 15

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom#4722]
#### User Story

> As a `[user role below]`:

dds designer

> I need to:

provide carousel design specs
> so that:

developers and adopters can reference the specs

#### Additional information
After the decision to develop [Carousel](https://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-carousel--default) as a separate component from [Card section – Carousel](https://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-card-section-carousel--default), we need design specs for Carousel on its own.

Meet with James and possibly a developer to define what is possible within the Carousel component. Things to consider:
- single image
- multiple images
- within light box

#### Acceptance criteria
 - [x] Design specs added to [Carousel component Box folder](https://ibm.ent.box.com/folder/121172699187?s=hmni7qhero9mf5rn593zij0tcay8h9dw)
 - [ ] Any updates to Carousel component defined in issue(s) for development
 - [x] Sign off from James and Wonil

**Comment by oliviaflory** [comment]
@wonilsuhibm @photodow if either of you have time to take a look at the [Carousel design specs](https://ibm.ent.box.com/file/763404121991) I'd appreciate any comments / questions.

The goal was to separate the Carousel component from the Card section – Carousel component, therefor does not have any text or headlines.

I started with examples of the Carousel with a general content area, then showed examples with generic Card component spanning all columns, then an example where the Carousel skips the first 4 columns, examples of how the Cards are displayed when cycling through a single desktop and tablet breakpoint. 

At the end I included wireframe-like examples of other potential layouts and card/image options for how Carousel could be used.

Let me know if you think we're missing anything. 

_Note: I haven't archived Jacob's design specs within the folder yet, so make sure you've got the right file:_ `component--carousel--design-specs--v1.0`
